conf/gce.yaml.template

```diff
@@ -0,0 +1,12 @@
+GCE:
+  # Google Provider as Compute Resource
+  # Project name in Google provider
+  PROJECT_ID: example-project-id
+  # Service Account email id which has compute admin permission
+  CLIENT_EMAIL: client@example.com
+  # client json Certificate path which is local path on satellite
+  CERT_PATH: /path/to/certificate.json
+  # Zones
+  ZONE: example-zone
+  # client certificate URL
+  CERT_URL: /path/to/certificate/url
```

Comment:
This should be mounted as a secret for good credentials practices, so this isn't the best suggestion for the comment.

Reply:
Do you want to make this a blocking suggestion?

Reply:
Sorry for the delay! No its fine, there are plenty of different approaches that can be taken, its not up to this comment to teach people security practices.